packages/core/src/definitions/styles.ts

```diff
@@ -120,42 +127,6 @@ export const builtInStyles: Partial<
     description: 'The spacing between the elements of the section',
     defaultValue: '0px',
   },
-  cfBackgroundImageUrl: {
```

Comment:
are we breaking anything backwards compat wise removing these?

Reply:
Yes, I'm pretty sure these are breaking changes and would require all components with background images and all image components to be removed and re-added to the canvas to get the correct variable definitions

Reply:
I think the background images are backwards compatible since the `cfBackgroundImageUrl` variable is staying in tact (I just moved it to the optional built in styles so it's not inherited on every custom component by default)

The `cfBackgroundImageAlignment` and `cfBackgroundImageScaling` variables have been removed and moved into the `cfBackgroundImageOptions` object, so these settings may revert to default options when the changes are released.